R/api.R

```diff
@@ -154,3 +155,28 @@ endpoint_kill <- function(runner) {
     pkgapi::pkgapi_state(runner = runner),
     returning = returning_json("Kill.schema"))
 }
+
+target_run_metadata <- function(runner) {
+  changelog <- runner$config$changelog$id[runner$config$changelog$public]
+  if (length(changelog) > 0) {
+    changelog <- vcapply(changelog, scalar, USE.NAMES = FALSE)
+  }
+  instances <- names(runner$config$database$source$instances)
+  if (length(instances) > 0) {
+    vcapply(instances, scalar, USE.NAMES = FALSE)
+  }
+  list(
+    instances_supported = scalar(length(instances) > 0),
```

Comment:
I think that the friendly name should come out here too (the one reported in slack messages etc).

Perhaps also version information while we're at it (orderly and orderly.server)